buildSrc/src/main/java/Deps.kt

```diff
@@ -88,6 +104,12 @@ object Deps {
     const val EqualsVerifier = "nl.jqno.equalsverifier:equalsverifier:$version"
   }
 
+  object GitHub {
+    object Kevinmost {
+      const val JUnitRetryRule = "com.github.kevinmost:junit-retry-rule:cbdd972d7c"
```

Comment:
I saw this unfortunately, but I didn't found any equivalent. The package is tested, stable, and do the job. So even if it is not anymore maintained, I do not see an issue here. In case of fix, we could fork it.

Reply:
Ah nice trick, thus that defeats the commit atomicity and introduce a commit that won't build, not sure that this is a good thing

Reply:
¯\_(ツ)_/¯